.github/workflows/verify.yaml

```
@@ -49,6 +49,31 @@ jobs:
       - name: Dependency analysis report
         run: cat build/reports/dependency-analysis/build-health-report.txt
 
+  OpenAPI-Definitions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '17'
+
+      - name: Save previous OpenAPI definition
+        run: yq -P 'sort_keys(..)' resources/openapi/openapi.yaml > openapi-git-sorted.yaml
+
+      - name: Generate OpenAPI definitions from code
+        run: |
+          ./gradlew resolve
+          ./gradlew mergeOpenApiFiles
+
+      # Can be used (and yq tasks removed) when https://github.com/kpramesh2212/openapi-merger-plugin/pull/11/files is merged
+      #- name: Check OpenAPI definitions match code
+      #  run: git diff --exit-code
+
+      - name: Check OpenAPI definitions match code
+        run: |
+          yq -P 'sort_keys(..)' resources/openapi/openapi.yaml > openapi-sorted.yaml
+          diff openapi-git-sorted.yaml openapi-sorted.yaml
```

adjusting accordingly, if we agree to take above suggestion.
```suggestion
          diff openapi-git-sorted.yaml openapi-ci-sorted.yaml
```